<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HealthCareWorkerRegistrationPage</class>
 <widget class="QWidget" name="HealthCareWorkerRegistrationPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Electronic Vaccination Data System</string>
  </property>
  <widget class="QPushButton" name="btn_hcwReg">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>S u b m i t</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>391</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Personal Information</string>
   </property>
   <widget class="QLineEdit" name="et_hcwName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Person</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="et_hcwSurname">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Doe</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Surname</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Role</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="et_hcwRole">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Doctor</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>391</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Account Information</string>
   </property>
   <widget class="QLineEdit" name="et_hcwPassreg1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>*****</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="et_hcwPassreg">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>*****</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Re-Enter Password</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lb_hcwmissingnfo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>All the Fields are compulsory, please fill in all of them.</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_hcwpassnotice">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>The passwords entered do not match.</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Your username will be [Name_Surname]</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_hcwErr">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Something went wrong please contact the system administrator</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
